indico/modules/events/sessions/util.py

```diff
@@ -0,0 +1,34 @@
+# This file is part of Indico.
```

Comment:
I think the last time we discussed about it we kind of decided on `operations.py` (since util is more for utility-like things, not for core functionality)